coffee_httpd/src/httpd/server.rs

```diff
@@ -51,6 +52,9 @@ pub async fn run_httpd<T: ToSocketAddrs>(
             .wrap_api()
             .service(swagger_api)
             .service(coffee_help)
+            .service(coffee_setup)
```

Comment:
```suggestion
            .service(coffee_setup)
```

I don't think this makes sense. The endpoint will be authenticated in some way so I guess the httpd makes sense only when core lightning is already started.

imho in the case of the httpd server needs to know the path of core lightning as an argument to the `coffee_httpd --cln-path=....` 

I do not have any use case in mind to support the setup endpoint